libqtile/widget/wttr.py

```diff
@@ -5,9 +5,12 @@
 
 class Wttr(GenPollUrl):
     """Display weather widget provided by wttr.in_.
+
```

Comment:
It looks like this is a commit full of whitespace only changes; if you want these can you squash them (and the message) into the previous commit?